src/redist/project.json

```diff
@@ -20,7 +21,8 @@
     "Microsoft.Build.Targets": "0.1.0-preview-00032-160824",
     "Microsoft.Build": "0.1.0-preview-00032-160824",
     "System.Runtime.Serialization.Xml": "4.1.0",
-    "NuGet.Build.Tasks": "3.6.0-beta.1.msbuild.1"
+    "NuGet.Build.Tasks": "3.6.0-beta.1.msbuild.1",
+	  "Microsoft.TestPlatform": "15.0.0-preview-*"
```

Comment:
We don't use `*` versions for packages that are outside of this repo.  Please use a specific version.